src/utils/clients.ts

```diff
@@ -65,7 +65,7 @@ export const setConnected = (() => {
 				try {
 					const response: Response = await request
 						.post({
-							url: `https://${BALENA_API_HOST}/services/vpn/client-${eventType}`,
+							url: `${BALENA_API_INTERNAL_HOST}/services/vpn/client-${eventType}`,
```

Comment:
Do you not need to set the protocol too? If it's implied and defaults to `http` I'd rather it was made explicit.

Reply:
Never mind, I just noticed protocol is included in the env var value, set a little further down.

Reply:
Yup it's set in `config.ts` rather than hardcoding the `https`/`http` in the rest of the code